chemreg/jsonapi/views.py

```diff
@@ -34,7 +34,15 @@ def create(self, request, *args, **kwargs):
 class ModelViewSet(
     ValidatePostParamsMixin, FormatRelatedFieldMixin, views.ModelViewSet
 ):
```

Comment:
@AScaduto , here's what I've implemented to only require Admin privileges when a `cid` is present in the post. Should work along with your ticket as well.